### DISCUSSION TITLE
DokAny v2 and case INSENSITIVE dir and tab-completion

### DISCUSSION THREAD

**Discussion by networkproblemsolver** [dokan-dev/dokany#1098] [General]
Hi, I am a user of Cryptomator, which currently supports the DokAny v1.

Even though Cryptomator is "case sensitive" when we use DokAny v1 we are able to get CASE INSENSITIVE results when doing "DIR N*" and "N"+TAB (tab-completion).

QUESTION:  Do you know if DokAny v2 will continue this feature of CASE INSENSITIVE dir and tab-completion???

(DokAny v1 is the only one that does; everyone else (WebDav, WinFsp) are case sensitive for DIR and TAB-COMPLETION.

Thx!

**Comment by Liryna** [comment]
Hi @networkproblemsolver ,

Dokan is by default case insensitive and can be case sensitive with the mount option [DOKAN_OPTION_CASE_SENSITIVE](https://github.com/dokan-dev/dokany/blob/da3078ddd5ad0bf2592670a41576fd4e297c3c5f/dokan/dokan.h#L109-L115).
~There were no change in V2 regarding this option.~

BUT it is true that during listing we ignore the option (make it case insensitive always) [here](https://github.com/dokan-dev/dokany/blob/master/dokan/directory.c#L415). It is probably a bug that should be fixed.

Edit:
I made a fix https://github.com/dokan-dev/dokany/commit/47ad17a8b44ed53e9b640269e88c6fa162854652 which is actually an issue introdued in v2. Thanks for your report !
